completion/git-machete.completion.zsh

```diff
@@ -217,7 +217,7 @@ __git_machete_github_subcommands() {
   local github_subcommands
   github_subcommands=(
     'anno-prs:annotate the branches based on their corresponding GitHub PR numbers and authors'
-    'checkout-prs:check out the given pull request locally'
+    'checkout-prs:check out the given pull requests locally'
```

Comment:
Pls add an issue to support the newly-added options of `checkout-prs` and reference it from here in a TODO